lib/invisible_captcha/controller_ext.rb

```diff
@@ -12,6 +12,15 @@ def detect_spam(options = {})
       if invisible_captcha?(options)
         on_spam_action(options)
       end
+      if invisible_captcha_threshold?(options)
```

Comment:
I did think the same but I wasn't sure how best to structure the tests to do it...and just wanted to get it working first so I'm all ears if you can suggest how to test both possible callback functions